Lib/test/test_os.py

```diff
@@ -770,7 +770,11 @@ def setUp(self):
         if support.can_symlink():
             os.symlink(os.path.abspath(t2_path), self.link_path)
             os.symlink('broken', broken_link_path, True)
-            self.sub2_tree = (sub2_path, ["link"], ["broken_link", "tmp3"])
+            if os.path.isdir(broken_link_path):
```

Comment:
Okay, I'll accept this into 3.4, Even Though It's In Security Fixes Only Mode etc etc etc.